mmv1/third_party/terraform/services/securitycenter/resource_scc_folder_big_query_export_config_test.go

```diff
@@ -0,0 +1,124 @@
+package securitycenter_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccSecurityCenterFolderBigQueryExportConfig_basic(t *testing.T) {
+	t.Parallel()
+
+	randomSuffix := acctest.RandString(t, 10)
+	dataset_id := "tf_test_" + randomSuffix
+	orgID := envvar.GetTestOrgFromEnv(t)
+
+	context := map[string]interface{}{
+		"org_id":              orgID,
+		"random_suffix":       randomSuffix,
+		"dataset_id":          dataset_id,
+		"big_query_export_id": "tf-test-export-" + randomSuffix,
+		"folder_name":         "tf-test-folder-name-" + randomSuffix,
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"random": {},
+			"time":   {},
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSecurityCenterFolderBigQueryExportConfig_basic(context),
+			},
+			{
+				ResourceName:            "google_scc_folder_scc_big_query_export.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"update_time"},
+			},
+			{
+				Config: testAccSecurityCenterFolderBigQueryExportConfig_update(context),
```

Comment:
It would be great to have a PlanCheck here to assert that the resource is updated-in-place. More details at: https://developer.hashicorp.com/terraform/plugin/testing/acceptance-tests/plan-checks/resource#expectresourceaction-plan-check

```suggestion
				Config: testAccSecurityCenterFolderBigQueryExportConfig_update(context),
				ConfigPlanChecks: resource.ConfigPlanChecks{
					PreApply: []plancheck.PlanCheck{
						plancheck.ExpectResourceAction("google_scc_folder_scc_big_query_export.default", plancheck.ResourceActionUpdate)
					},
				},
```

Reply:
Addressed.